src/js/deepLinks.ts

```diff
@@ -29,9 +30,63 @@ for (let dr in deepRoutes) deepRouter.add(dr, deepRoutes[dr]);
 
 export default {
   init() {
-    // open games from external links with url scheme
-    window.handleOpenURL = function(url: string) {
+    // open links that follow the custom url scheme
+      window.handleOpenURL = function(url: string) {
```

Comment:
Why keeping both `deepRouter` and `universalLinks` routers? If I understand correctly, universalLinks is a replacement of the older solution. It seems it will create conflicts if both stays.